pkg/agent/openflow/multicast.go

```diff
@@ -91,6 +97,13 @@ func (f *featureMulticast) replayFlows() []*openflow15.FlowMod {
 	return getCachedFlowMessages(f.cachedFlows)
 }
 
+func (f *featureMulticast) multicastFlexibleIPAMGroup(groupID binding.GroupIDType) binding.Group {
+	group := f.bridge.NewGroupTypeAll(groupID)
+	group = group.Bucket().LoadRegMark(OutputToOFPortRegMark).Output(f.hostOFPort).Done()
+	group = group.Bucket().LoadRegMark(OutputToOFPortRegMark).Output(f.uplinkPort).Done()
```

Comment:
Will it break the metrics collections in the later flows if we output packet directly here?

Reply:
Updated. This func is removed. Do you mean using `output:uplink,hostOFPort` as you suggested? It won't break metrics collections.